Library/.rubocop.yml

```diff
@@ -2,6 +2,7 @@ AllCops:
   Exclude:
     - 'Homebrew/vendor/**/*'
     - 'Homebrew/test/vendor/**/*'
+    - 'Homebrew/compat/**/*'
```

Comment:
Ok. Can we ignore the specific rule and add a comment explaining why instead, maybe?